src/components/PageBanner/_PageBanner.scss

```diff
@@ -1,7 +1,6 @@
 .coa-PageBanner {
-  background-color: $coa-color-gray-lighter;
-  background-position: center center;
-  background-size: cover;
+  object-fit: cover;
```

Comment:
same as above re: polyfill

Reply:
#243 again for it

Reply:
@briaguya could you port #243 over to https://github.com/cityofaustin/techstack/issues with the `Technical Debt` label?

Reply:
We're logging issues in techstack now and trying to close out things in janis so tasks for alpha can be managed in one place. closed this one for this https://github.com/cityofaustin/techstack/issues/419